blockchain_exploration/__init__.py

```diff
@@ -1 +1 @@
-VERSION = "1.4.0"
+VERSION = "1.8.1"
```

Comment:
Minor, but looks confusing when we see the wrong version get installed in build logs.